### PR TITLE
Purchase_received_qty

### PR DESCRIPTION
**Pull request by TimLai125**

### CODE REVIEW

**Comment by TimLai125** [comment]
The module is good to review

File: purchase_received_qty_adj/models/purchase_order_line.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields, api
+import odoo.addons.decimal_precision as dp
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    qty_received_adj = fields.Float(
```
Comment (yostashiro) [inline purchase_received_qty_adj/models/purchase_order_line.py]: Change this to `qty_received_net`.

File: purchase_received_qty_adj/models/purchase_order_line.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        string='Received Qty',
+        compute='_compute_qty_received_adj',
+        digits=dp.get_precision('Product Unit of Measure'),
+        copy=False,
+        store=True,
+        default=0.0
```
Comment (yostashiro) [inline purchase_received_qty_adj/models/purchase_order_line.py]: `help="This field shows the net received qty (Received Qty minus Returned Qty)"`

File: purchase_received_qty_adj/models/purchase_order_line.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+from odoo import models, fields, api
+import odoo.addons.decimal_precision as dp
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    qty_received_adj = fields.Float(
+        string='Received Qty',
```
Comment (yostashiro) [inline purchase_received_qty_adj/models/purchase_order_line.py]: `string='Received Qty (Net)'`

File: purchase_received_qty_adj/views/purchase_order_view.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//field[@name='order_line']/tree/field[@name='qty_refunded']"
+                    position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//field[@name='order_line']/tree/field[@name='qty_received']"
+                    position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
```
Comment (yostashiro) [inline purchase_received_qty_adj/views/purchase_order_view.xml]: As we just talked, we will not hide this field.

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]

**Pull request merged**